src/Tvheadend.cpp

```diff
@@ -1827,7 +1827,7 @@ void CTvheadend::ParseChannelAddOrUpdate ( htsmsg_t *msg, bool bAdd )
         {
           if (!strcmp(str, "Radio"))
             channel.SetType(PVR_RECORDING_CHANNEL_TYPE_RADIO);
-          else
+          else if (!strcmp(str, "SDTV") || !strcmp(str, "HDTV"))
```

Comment:
"UHDTV" was added with htspv25, this code only gets called when htsp version < 25.